packages/grafana-ui/src/components/ColorPicker/NamedColorsGroup.tsx

```diff
@@ -18,41 +20,68 @@ const NamedColorsGroup: FunctionComponent<NamedColorsGroupProps> = ({
   ...otherProps
 }) => {
   const primaryShade = hue.shades.find((shade) => shade.primary)!;
+  const label = upperFirst(hue.name);
+  const styles = useStyles2(getStyles);
 
   return (
-    <div {...otherProps} style={{ display: 'flex', flexDirection: 'column' }}>
-      {primaryShade && (
-        <ColorSwatch
-          key={primaryShade.name}
-          isSelected={primaryShade.name === selectedColor}
-          variant={ColorSwatchVariant.Large}
-          color={primaryShade.color}
-          label={upperFirst(hue.name)}
-          onClick={() => onColorSelect(primaryShade.name)}
-        />
-      )}
-      <div
-        style={{
-          display: 'flex',
-          marginTop: '8px',
-        }}
-      >
-        {hue.shades.map(
-          (shade) =>
-            !shade.primary && (
-              <div key={shade.name} style={{ marginRight: '4px' }}>
-                <ColorSwatch
-                  key={shade.name}
-                  isSelected={shade.name === selectedColor}
-                  color={shade.color}
-                  onClick={() => onColorSelect(shade.name)}
-                />
-              </div>
-            )
+    <div className={styles.colorRow} style={{}}>
+      <div className={styles.colorLabel}>{label}</div>
+      <div {...otherProps} className={styles.swatchRow}>
+        {primaryShade && (
+          <ColorSwatch
+            key={primaryShade.name}
+            aria-label={primaryShade.name}
+            isSelected={primaryShade.name === selectedColor}
+            variant={ColorSwatchVariant.Large}
+            color={primaryShade.color}
+            onClick={() => onColorSelect(primaryShade.name)}
+          />
         )}
+        <div className={styles.swatchContainer}>
+          {hue.shades.map(
+            (shade) =>
+              !shade.primary && (
+                <div key={shade.name} style={{ marginRight: '4px' }}>
+                  <ColorSwatch
+                    key={shade.name}
+                    aria-label={shade.name}
+                    isSelected={shade.name === selectedColor}
+                    color={shade.color}
+                    onClick={() => onColorSelect(shade.name)}
+                  />
+                </div>
+              )
+          )}
+        </div>
       </div>
     </div>
   );
 };
 
 export default NamedColorsGroup;
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    colorRow: css`
+      display: grid;
+      grid-template-columns: 25% 1fr;
+      grid-column-gap: ${theme.spacing(2)};
+      padding: ${theme.spacing(1, 0)};
+
+      &:hover {
+        background: ${theme.colors.background.secondary};
+      }
```

Comment:
i would add `align-items: center;` here else the text is top aligned which looks a little weird. idk what the design has though? 🤔

Reply:
it's inline with the design though.